change/@ni-nimble-components-9a663b6a-ecb5-4698-bd2c-77cb3281255a.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
FYI I started review with individual comments accidentally but will bundle the rest of the comments in a review.

Reply:
Sorry I did not finish the review today but expect to tomorrow